.github/workflows/test-opencast-build.yml

```diff
@@ -20,6 +20,8 @@ on:
 
```

Comment:
I did not go with that approach because I was under the impression that `branches-ignore` filters by target branch (e.g. develop instead of dependabot/**). But maybe I'm wrong about that, not a github actions expert at all. I would also prefer a filter at event level, so if @ziegenberg 's proposal works let's go with that.

Reply:
As we are filtering here on push events and not on pull request events, the target branch is `dependabot/**` and it should work. As long as I'm not completely mistaken...

Reply:
I filed an alternative PR here: #4207